packages/react/src/components/Checkbox/Checkbox.js

```diff
@@ -19,6 +19,7 @@ const Checkbox = React.forwardRef(function Checkbox(
     onChange,
     indeterminate,
     hideLabel,
+    readOnly,
```

Comment:
Even though checkboxes don't support the standard attribute `readonly`, what are your thoughts on naming this `readonly` (lowercase) for consistency with the other inputs that will use `readonly`?

Reply:
My preference would be to use lowercase `readonly` everywhere as we are using it for the standards defined purpose. This is simply repeating the pattern used in TextInput for consistency.

Reply:
Unfortunately React says not to use 'readonly' https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes